pkgs/terraform-provider-huaweicloud.nix

```diff
@@ -19,5 +19,7 @@ buildGoModule rec {
 
   postInstall = "mv $out/bin/terraform-provider-huaweicloud{,_v${version}}";
 
+  patches = [ ./terraform-provider-huaweicloud.patch ];
```

Comment:
Is this patch submitted upstream ?

Reply:
The pull request has been merged but there is no release yet. Thas is why i had to add it like that for the moment

Reply:
si il force push ou squash la pr ça va poser des pr non?